taglib/src/main/java/com/reevoo/taglib/AbstractReevooMarkClientTag.java

```diff
@@ -119,7 +107,20 @@ protected void setClient(ReevooMarkClient client) {
             // with the parameter "reviews" instead of the parameter "per_page".
             queryStringParams.put("reviews",this.numberOfReviews);
         }
+        if (this.dynamicAttrs != null) {
+          for (Map.Entry<String,String> entry : this.dynamicAttrs.entrySet()) {
```

Comment:
yes the compiler does that for you, one of the few things java does to help make your code a little less verbose than it already is.